internal/executor/executor.go

```diff
@@ -32,6 +34,13 @@ type RealExecutor struct {
 }
 
 func (e *RealExecutor) Execute(output io.Writer, workingDir string, name string, args ...string) error {
+	if flags.DryRun {
+		msg := fmt.Sprintf("Dry-run mode: %s %s. Working dir: %s", name, args, workingDir)
```

Comment:
Based on the example output `"Dry-run mode: git [checkout -b some_branch]. Working dir: work/org/repo1"`: I think it would be good if the command + args could look exactly as a user would type it, to avoid confusion. 

Perhaps we could:
* put the working dir on a separate line - this would make it unambiguous what is the command + args, especially if the output is long.
* eliminate the `[`/`]` characters. Having them visible here might make users (incorrectly) think that we're going to execute an invalid shell command.

Reply:
Having said this 😄...

I wonder if we should:
* combine this with line 49 below: `_, err := fmt.Fprintln(output, "Executing:", name, summarizedArgs(args), "in", workingDir)`. All it would take is to conditionally change `Executing` to `Dry-run mode, would execute`.
* `summarizedArgs` would probably be a good function to use _anyway_ (it hides very long args, like PR titles/descriptions). `summarizedArgs` could be made better by hiding the `[`/`]` characters around the array.

Reply:
Done in https://github.com/Skyscanner/turbolift/pull/88/commits/89a68e308ef04702b6a1a0859764d960e9a75a53